pkg/gather/clusterconfig/authentications.go

```diff
@@ -1,3 +1,4 @@
+//nolint: dupl
```

Comment:
What is the reason for this pls?

Reply:
code duplication, some parts of the code is duplicated mostly the place where the clients get created, but IMO its cleaner this way, (for backporting)

Reply:
Ahh OK thanks. I needed to run the linting to understand it ... :)